front/src/modules/auth/states/displayGoogleLogin.ts

```diff
@@ -0,0 +1,6 @@
+import { atom } from 'recoil';
+
+export const displayGoogleLogin = atom<boolean>({
```

Comment:
Tbd how this will evolve when we have a great number of feature toggles in the future.
For now I think your approach is the best and cleanest, and we can merge like this in my opinion.
But one day we'll probably 50+ state (feature flags of all kinds), in that case maybe a single clientConfig state with an array would work out best

Reply:
Thanks for the suggesetion, i will work on this.